pegasus/sites.v3/code.org/views/index_video_reveal.haml

```diff
@@ -1,8 +1,16 @@
+- fallback_player ||= false
+
 %div{style: "text-align:right; overflow:hidden"}
   %div{style: "width:100%; height: 100%; margin: 0 auto;"}
     .videodiv{style: "position:relative;"}
       %img{style: "display:block; width:100%; height:auto", src: "/images/16x9.png"}/
-      %iframe{style: "position:absolute; top: 0; left: 0; width: 100%; height: 100%", src: "https://www.youtubeeducation.com/embed/#{video_code}?iv_load_policy=3&rel=0&autohide=1&showinfo=0&enablejsapi=1", frameborder: "0", allowfullscreen: true}
+
+      - if fallback_player
+        -# either youtube or fallback video player inserted here
+        .insert_video_player{:data=>{:video_code=>video_code, :download_path=>download_path}}
+      - else
+        -# regular youtube video player
+        %iframe{style: "position:absolute; top: 0; left: 0; width: 100%; height: 100%", src: "https://www.youtube.com/embed/#{video_code}?iv_load_policy=3&rel=0&autohide=1&showinfo=0&enablejsapi=1", frameborder: "0", allowfullscreen: true}
```

Comment:
Is there a way we could de-duplicate this line and the one found within the `.insert_video_player` swapper code? One way could be, e.g., have this line use `.insert_video_player` as well, but with some `data`->`use_fallback` or `do_not_use_fallback` parameter (or just by detecting it not including the `download_path` parameter)